### PR TITLE
Convert BreadcrumbComponent, ReleaseNoteComponent and ReleaseNotesView to Vue 3.

### PR DESCRIPTION
**Pull request by BrianMaki**
# Implements [AB#15765](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/15765), #15774. #15775

## Description

Converted following to Vue 3:

- BreadcrumbComponent
- ReleaseNoteComponent
- ReleaseNotesView

**Desktop:**

Screenshot 2023-06-26 at 12 53 48 PM

**Iphone 6:**

Screenshot 2023-06-26 at 1 11 10 PM

**iPhone X:**

Screenshot 2023-06-26 at 1 11 23 PM

**Changes to styling in BreadcrumbComponent:**

Screenshot 2023-06-26 at 12 55 07 PM

Screenshot 2023-06-26 at 12 59 19 PM

**Changes to styling in ReleaseNoteComponent:**

Screenshot 2023-06-26 at 1 00 50 PM

Screenshot 2023-06-26 at 1 02 22 PM

## Testing

- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [x] Not Required

## UI Changes

Yes

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

**Review by mk08sh — COMMENTED** [review]
👍

**Review by BronzBierd — APPROVED** [review]

File: Apps/WebClient/src/NewClientApp/src/components/ReleaseNoteComponent.vue [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+<template>
+    <div>
+        <v-row>
+            <v-col class="text-body-1 font-weight-bold">
+                {{ date }} – {{ version }} – {{ title }}
+            </v-col>
+        </v-row>
+        <v-row v-if="hasSlot" class="mt-1 mb-2">
+            <v-col class="px-3 py-0">
```
Comment (BronzBierd) [inline Apps/WebClient/src/NewClientApp/src/components/ReleaseNoteComponent.vue:25]: This doesn't affect my review, but if you only want to push the alignment one direction I would bother applying padding to both sides, so `pl-3` would have been fine.

File: Apps/WebClient/src/NewClientApp/src/views/ReleaseNotesView.vue [hunk 2]
```diff
@@ -0,0 +1,1487 @@
...
+        to: "/release-notes",
+        active: true,
+        dataTestId: "breadcrumb-release-notes",
+    },
+];
+</script>
+
+<template>
+    <v-container>
+        <div>
```
Comment (MikeLyttle) [inline Apps/WebClient/src/NewClientApp/src/views/ReleaseNotesView.vue:19]: This wrapping `<div>` is not doing anything & can be removed.

File: Apps/WebClient/src/NewClientApp/src/components/ReleaseNoteComponent.vue [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+                <slot />
+            </v-col>
+        </v-row>
```
Comment (MikeLyttle) [inline Apps/WebClient/src/NewClientApp/src/components/ReleaseNoteComponent.vue]: Rows and columns aren't needed & can be replaced with divs.

File: Apps/WebClient/src/NewClientApp/src/components/navmenu/BreadcrumbComponent.vue [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+<template>
+    <v-breadcrumbs
+        v-if="displayBreadcrumbs"
+        data-testid="breadcrumbs"
+        class="pt-0 pb-0"
+        aria-label="Breadcrumb Nav"
+    >
+        <template v-for="(item, index) in allBreadcrumbItems" :key="item.text">
+            <v-breadcrumbs-item
+                :to="item.to"
```
Comment (MikeLyttle) [inline Apps/WebClient/src/NewClientApp/src/components/navmenu/BreadcrumbComponent.vue:45]:
```suggestion
               :to="!item.active ? item.to : undefined"
```

Without this change, the currently displayed page will be a clickable link.

File: Apps/WebClient/src/NewClientApp/src/components/ReleaseNoteComponent.vue (hunk 1 above)
Comment (BrianMaki) [reply Apps/WebClient/src/NewClientApp/src/components/ReleaseNoteComponent.vue]: n/a anymore

**Review by MikeLyttle — APPROVED** [review]

**Pull request merged**